pkg/templates/templates_generated.go

```diff
@@ -7447,7 +7447,8 @@ func windowsWindowscsehelperPs1() (*asset, error) {
 }
 
 var _windowsWindowscsehelperTestsPs1 = []byte(`BeforeAll {
```

Comment:
interesting. Could we exclude _windowsWindowscsehelperTestsPs1 in templates_generated.go since it is only for ut?

Reply:
If we exclude _windowsWindowscsehelperTestsPs1, we would also need to exclude windowscsehelper.tests.ps1 in templates_generated.go. Is it ok?

Reply:
yes. I think that we should not include the test file in the template. It was a mistake.

Reply:
Thanks for the comment. Correct the mistake now.